prime-router/docs/getting-started/psql-clients.md

```diff
@@ -0,0 +1,20 @@
+# PostgreSQL Client Tools
+
+There are a variety of options to interact with your PostgreSQL instance:
+
+* DBeaver Community Edition can be downloaded [from here](https://dbeaver.io/download/). It is a locally installed, graphical tool that lets you connect to a variety of database systems, including PostgreSQL with [very extensive documentation wiki](https://github.com/dbeaver/dbeaver/wiki).
+* pgAdmin: if you want to install pgAdmin, we recommend running it as a docker container attached to the `prime-router_build` network so that it can talk to the `postgresql` container used by ReportStream.
+* `psql`: the command-line tool for text-based interactions with PostgreSQL, this should come with your PostgreSQL installation
+
+    ```
+    $ psql --host localhost --dbname prime_data_hub --user prime
```

Comment:
You are giving a PSQL class here.  The purpose of this document should be to get the user up and running, so let them do their own learning somewhere else if they want more.  This will make this doc a bit simpler too.